test/data/testBasis.py

```diff
@@ -46,30 +52,35 @@ def testsizeofbasis(self):
         """Is the basis set the correct size?"""
 
         total = 0
-        multiple = self.multiple
-        if self.spherical:
-            multiple = self.multiple_spher
-
+        multiple = self.multiple_spher if self.spherical else self.multiple
         for atom in self.data.gbasis:
-
-            for fns in atom:
-
-                 # Add 3 for P, 5 or 6 for D, and so forth.
-                ftype = fns[0]
+            for (ftype, contraction) in atom:
                 total += multiple[ftype]
 
         self.assertEquals(self.data.nbasis, total)
 
+    def testcontractions(self):

```

Comment:
I added them in a PR to your PR :) I think here: ce70756a516f76fa82caeef4b41c8dbc96484325